src/Command/BaseCommand.php

```diff
@@ -87,7 +122,7 @@ public function getArguments()
      * @return void
      * @throws InvalidCommandArgumentException
      */
-    public function setArguments(array $arguments)
+    public function setArguments(array $arguments = [])
```

Comment:
Why would this method be called without arguments? I see no point in adding a default value, that is actually a way for this method to be documented as argument mandatory